array-manipulation-1/src/tail.js

```diff
@@ -0,0 +1,22 @@
+/* exported tail */
+function tail(array) {
+  var newArray = [];
+  delete array[0];
```

Comment:
Oh no, this is making changes to the original array, you should just delete this line.